proto/oracle/v1/oracle.proto

```diff
@@ -261,18 +308,19 @@ message Params {
   uint64 inactive_penalty_duration = 8;
 }
 
-// PendingResolveList
+// PendingResolveList is a list of requests that are waiting to be resolved by validators
```

Comment:
Requests would be resolved by any nodes and the results would be stored locally.